test/nodes/test_cost_tracker.py

```
@@ -0,0 +1,568 @@
+        )
+        tracker.track(entry)
+        answer.metadata['cost_usd'] = entry['cost_usd']
+        answer.metadata['cumulative_cost_usd'] = tracker.get_total()
+
+        assert 'cost_usd' in answer.metadata
+        assert answer.metadata['cost_usd'] > 0
+        assert answer.metadata['cumulative_cost_usd'] > 0
+
+    def test_deep_copy_prevents_mutation(self):
+        """Deep copying the answer prevents upstream mutation."""
+        tracker, answer = self._build_instance()
```

Ruff (0.15.7)

[warning] Unpacked variable `tracker` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)